In [1]:
import pandas as pd
import numpy as np

In [3]:
df_actor = pd.read_csv("actor.csv")

In [6]:
df_actor.describe()

Age
count  32.000000
mean   44.718750
std    10.264987
min    31.000000
25%    37.750000
50%    42.500000
75%    48.750000
max    76.000000

**Conclusiones (resumen)**:

a. Existen 32 observaciones.

b. La edad media es 45 años.

c. MÍN.: 31 | Q1: 37.75 | MEDIAN: 42.5 | Q2: 48.75 | MÁX.: 76

d. Antes de los 44 años.

e. (31 - 76) AÑOS : 45 años de rango.

f. IQR:  48.75 - 37.75 = 11 AÑOS
